frontend/src/components/ModelBox/ModelSelection.tsx

```diff
@@ -22,6 +29,10 @@ function ModelSelection({ chatModelOptions }: { chatModelOptions: string[] }) {
 		if (!isSettingModel && selectedModel) {
 			const currentSelectedModel = selectedModel;
 			console.log(`selected model: ${currentSelectedModel}`);
+			addChatMessage({
+				type: CHAT_MESSAGE_TYPE.INFO,
+				message: `GPT model changed to: ${currentSelectedModel}`,
+			});
```

Comment:
You should import and use `addInfoMessage` instead. That'll sort out the refresh problem